Python/Python-ast.c

```diff
@@ -3954,6 +3958,11 @@ ast2obj_arguments(void* _o)
     if (_PyObject_SetAttrId(result, &PyId_args, value) == -1)
         goto failed;
     Py_DECREF(value);
+    value = ast2obj_list(o->posonlyargs, ast2obj_arg);
+    if (!value) goto failed;
```

Comment:
Per PEP 7, you should have curly braces after all control statements, even where optional.

It may be that I'm mis-reading this diff and you're not actually *adding* non-compliant code, but I think it's worth a refactor-as-you-go fix for this.

Reply:
This file is automatically generated